recipes/ogre/1.x/conanfile.py

```diff
@@ -0,0 +1,320 @@
+        tools.rmdir(os.path.join(self.package_folder, "share"))
+        self._create_cmake_module_variables(
+            os.path.join(self.package_folder, self._module_file_rel_path),
+            tools.Version(self.version)
+        )
+        
+        
+    @staticmethod
+    def _create_cmake_module_variables(module_file, version):
+        content = textwrap.dedent("""\
+            set(OGRE_PREFIX_DIR ${{CMAKE_CURRENT_LIST_DIR}}/../..)
+            set(OGRE{major}_VERSION_MAJOR {major})
+            set(OGRE{major}_VERSION_MINOR {minor})
+            set(OGRE{major}_VERSION_PATCH {patch})
+            set(OGRE{major}_VERSION_STRING "{major}.{minor}.{patch}")
+
+            set(OGRE_MEDIA_DIR "${{OGRE_PREFIX_DIR}}/share/OGRE/Media")
+            set(OGRE_PLUGIN_DIR "${{OGRE_PREFIX_DIR}}/lib/OGRE")
+            set(OGRE_CONFIG_DIR "${{OGRE_PREFIX_DIR}}/share/OGRE") 
+        """.format(major=version.major, minor=version.minor, patch=version.patch))
+        tools.save(module_file, content)
+
+
+    @property
+    def _components(self):
+        pkg_name = "OGRE"
+        include_prefix = os.path.join("include", "OGRE")
+        components = {
+            "OgreMain":  {"requires" : ["boost::boost", "cppunit::cppunit", "freeimage::freeimage", "openexr::openexr","freetype::freetype", 
+                                        "sdl::sdl", "tbb::tbb", "xorg::xorg", "zlib::zlib", "zziplib::zziplib", "poco::poco","glu::glu", "egl::egl"], 
+                            "libs": ["OgreMain"], "include": [include_prefix]},
+            "Bites":  {"requires" : ["OgreMain", "Overlay"], "libs": ["OgreBites"], "include": ["include", include_prefix, f"{include_prefix}/Bites"]},
+            "HLMS" :  {"requires" : ["OgreMain"], "libs": ["OgreHLMS"], "include": ["include", include_prefix, f"{include_prefix}/HLMS"]},
+            "MeshLodGenerator" :  {"requires" : ["OgreMain"], "libs": ["OgreMeshLodGenerator"], "include": ["include", include_prefix, f"{include_prefix}/MeshLodGenerator"]},
+            "Overlay" :  {"requires" : ["OgreMain"], "libs": ["OgreOverlay"], "include": ["include", include_prefix, f"{include_prefix}/Overlay"]},
+            "Paging" :  {"requires" : ["OgreMain"], "libs": ["OgrePaging"], "include": ["include", include_prefix, f"{include_prefix}/Paging"]},
+            "Property" :  {"requires" : ["OgreMain"], "libs": ["OgreProperty"], "include": ["include", include_prefix, f"{include_prefix}/Property"]},
+            "RTShaderSystem" :  {"requires" : ["OgreMain"], "libs": ["OgreRTShaderSystem"], "include": ["include", include_prefix, f"{include_prefix}/RTShaderSystem"]},
+            "Terrain" :  {"requires" : ["OgreMain"], "libs": ["OgreTerrain"], "include": ["include", include_prefix, f"{include_prefix}/Terrain"]},
+            "Volume" :  {"requires" : ["OgreMain"], "libs": ["OgreVolume"], "include": ["include", include_prefix, f"{include_prefix}/Volume"]}
+            
+        }
+
+        if self.options.ogre_static:
```

Comment:
```suggestion
        if not self.options.shared:
```
this should fix the CI failure